src/main/kotlin/nl/info/zac/app/identity/IdentityRestService.kt

```diff
@@ -39,10 +39,16 @@ class IdentityRestService @Inject constructor(
     @Path("groups")
     fun listGroups(): List<RestGroup> = identityService.listGroups().toRestGroups()
 
+    /**
+     * Once the PABC feature flag has been removed, this should be refactored to take the zaaktype 'omschrijving' field
+     * instead of the zaaktype UUID.
+     * This is because in the PABC group authorisation is done on zaaktype and not on a
+     * specific zaaktype 'version'.
+     */
     @GET
     @Path("groups/zaaktype/{zaaktypeUuid}")
-    fun listGroupsForZaaktypeUuid(@PathParam("zaaktypeUuid") zaaktypeUuid: UUID): List<RestGroup> =
-        identityService.listGroupsForZaaktypeUuid(zaaktypeUuid).toRestGroups()
+    fun listGroupsForBehandelaarRoleAndZaaktypeUuid(@PathParam("zaaktypeUuid") zaaktypeUuid: UUID): List<RestGroup> =
```

Comment:
@infonl-marcel @xiduzo double check for you guys: the `/identity/groups/zaaktype/{zaaktypeUuid}` endpoint is always used to retrieve a list of group so that a specific zaak may be assigned to this group, correct? 

In the new IAM situation group authorisation is always done on ZAC application role and zaaktype. As application role I think we need to use `behandelaar` here because zaken may only be assigned to groups (and therefore employees) with the role behandelaar.